proxies/java/src/main/java/com/dvc_harness/proxy/data/DataStore.java

```diff
@@ -0,0 +1,23 @@
+package com.dvc_harness.proxy.data;
+
+import java.util.Hashtable;
+
+import com.devcycle.sdk.server.cloud.api.DVCCloudClient;
+import com.devcycle.sdk.server.local.api.DVCLocalClient;
+import com.devcycle.sdk.server.common.model.*;
+
+class DataStore {
+    public static Hashtable<String, DVCLocalClient> LocalClients;
+    public static Hashtable<String, DVCCloudClient> CloudClients;
+
+    public static Hashtable<String, User> Users;
+    public static Hashtable<String, String> Commands;
+
+    public DataStore()
```

Comment:
is this a constructor for an instance of this? don't you want to use the static properties instead? This constructor would then need to be static

Reply:
this was a complete afterthought on the rest of the PR

Reply:
just threw it together because I saw it in the c# scaffolding, will need to figure out how this should actually look

Reply:
might be better to just remove it then if this isn't intended to be used as-is, lest somebody try and use it

Reply:
yea I think just making it static should do what we want